### PR TITLE
[Word] Snippets for reqt. set 1.6

### PR DESCRIPTION
**Pull request by ElizabethSamuel-MSFT**

### CODE REVIEW

File: samples/word/50-document/manage-tracked-changes.yaml [hunk 1]
```diff
@@ -0,0 +1,218 @@
...
+            <button id="setup" class="ms-Button">
+            <span class="ms-Button-label">Add sample text</span>
+          </button>
+        </section>
+
+        <section class="samples ms-font-m">
+            <h3>Try it out</h3>
+            <button id="get-all-tracked-changes" class="ms-Button">
+            <span class="ms-Button-label">Get all tracked changes</span>
+          </button>
```
Comment (samantharamon) [inline samples/word/50-document/manage-tracked-changes.yaml:169]:
```suggestion
            <button id="get-all-tracked-changes" class="ms-Button">
                <span class="ms-Button-label">Get all tracked changes</span>
            </button>
```

File: samples/word/50-document/manage-tracked-changes.yaml [hunk 2]
```diff
@@ -0,0 +1,218 @@
...
+            <button id="accept-all-tracked-changes" class="ms-Button">
+                <span class="ms-Button-label">Accept all tracked changes</span>
+            </button>
+            <p>First, press the <b>Add sample text</b> button.
+                <button id="reject-all-tracked-changes" class="ms-Button">
+                <span class="ms-Button-label">Reject all tracked changes</span>
+              </button>
+            </p>
```
Comment (samantharamon) [inline samples/word/50-document/manage-tracked-changes.yaml:177]:
```suggestion
            <p>First, choose the <b>Add sample text</b> button.
                <button id="reject-all-tracked-changes" class="ms-Button">
                    <span class="ms-Button-label">Reject all tracked changes</span>
                </button>
            </p>
```

File: samples/word/50-document/manage-tracked-changes.yaml [hunk 3]
```diff
@@ -0,0 +1,218 @@
...
+            <p>First, press the <b>Add sample text</b> button.
+                <button id="get-next-tracked-change" class="ms-Button">
+            <span class="ms-Button-label">Get the next (2nd) tracked change</span>
+            </button>
+            </p>
```
Comment (samantharamon) [inline samples/word/50-document/manage-tracked-changes.yaml:182]:
```suggestion
            <p>First, choose the <b>Add sample text</b> button.
                <button id="get-next-tracked-change" class="ms-Button">
                    <span class="ms-Button-label">Get the next (second) tracked change</span>
                </button>
            </p>
```

File: samples/word/50-document/manage-tracked-changes.yaml [hunk 4]
```diff
@@ -0,0 +1,218 @@
...
+            await callback();
+          } catch (error) {
+            // Note: In a production add-in, you'd want to notify the user through your add-in's UI.
+            console.error(error);
+          }
+        }
+    language: typescript
+template:
+    content: |-
+        <section class="ms-font-m">
```
Comment (samantharamon) [inline samples/word/50-document/manage-tracked-changes.yaml:154]: Suggestion: Consider grouping similar operations together. For example:

- Get all tracked changes
- Get the next (second) tracked change
- Get the first tracked change's range
- Accept the first tracked change
- Reject the first tracked change

Choose the **Add sample text** button before testing the following options.
- Accept all tracked changes
- Reject all tracked changes

File: samples/word/50-document/manage-tracked-changes.yaml [hunk 5]
```diff
@@ -0,0 +1,218 @@
...
+          </button>
+            <button id="accept-all-tracked-changes" class="ms-Button">
+                <span class="ms-Button-label">Accept all tracked changes</span>
+            </button>
+            <p>First, press the <b>Add sample text</b> button.
+                <button id="reject-all-tracked-changes" class="ms-Button">
+                <span class="ms-Button-label">Reject all tracked changes</span>
+              </button>
+            </p>
+            <p>First, press the <b>Add sample text</b> button.
```
Comment (samantharamon) [inline samples/word/50-document/manage-tracked-changes.yaml:178]:
```suggestion
            <p>First, choose the <b>Add sample text</b> button.
```

File: samples/word/25-paragraph/onchanged-event.yaml [hunk 6]
```diff
@@ -0,0 +1,128 @@
+order: 10
+id: word-paragraph-onchanged-event
+name: On changing content in paragraphs
+description: 'Registers, triggers, and deregisters onParagraphChanged event that tracks when content is changed in paragraphs.'
```
Comment (samantharamon) [inline samples/word/25-paragraph/onchanged-event.yaml:4]:
```suggestion
description: 'Registers, triggers, and deregisters the onParagraphChanged event that tracks when content is changed in paragraphs.'
```

File: samples/word/25-paragraph/onchanged-event.yaml [hunk 7]
```diff
@@ -0,0 +1,128 @@
...
+        }
+
+        async function deregisterEventHandler() {
+          await Word.run(eventContext.context, async (context) => {
+            eventContext.remove();
+            await context.sync();
+          });
+
+          eventContext = null;
+          console.log("Removed event handler that was tracking when content is changed in paragraphs.");
```
Comment (samantharamon) [inline samples/word/25-paragraph/onchanged-event.yaml:52]:
```suggestion
          console.log("Removed event handler that was tracking content changes in paragraphs.");
```

File: samples/word/25-paragraph/onchanged-event.yaml [hunk 8]
```diff
@@ -0,0 +1,128 @@
...
+            <span class="ms-Button-label">Add paragraphs</span>
+          </button>
+        </section>
+
+        <section class="samples ms-font-m">
+            <h3>Try it out</h3>
+            <button id="register-event-handler" class="ms-Button">
+            <span class="ms-Button-label">Register event handler</span>
+          </button>
+            <p><div class="ms-font-m">Within a paragraph, make a change in the text.</div><div class="ms-font-m">Get the changed paragraph by ID.</div>
```
Comment (samantharamon) [inline samples/word/25-paragraph/onchanged-event.yaml:92]:
```suggestion
            <p><div class="ms-font-m">Within a paragraph, make a change in the text. Then, from the console, copy the paragraph ID where content was changed and paste into the following field.</div><div class="ms-font-m">Get the changed paragraph by ID.</div>
```

File: samples/word/25-paragraph/ondeleted-event.yaml [hunk 9]
```diff
@@ -0,0 +1,133 @@
+order: 11
+id: word-paragraph-ondeleted-event
+name: On deleting paragraphs
+description: 'Registers, triggers, and deregisters onParagraphDeleted event that tracks the removal of paragraphs.'
```
Comment (samantharamon) [inline samples/word/25-paragraph/ondeleted-event.yaml:4]:
```suggestion
description: 'Registers, triggers, and deregisters the onParagraphDeleted event that tracks the removal of paragraphs.'
```

File: samples/word/25-paragraph/onadded-event.yaml [hunk 10]
```diff
@@ -0,0 +1,128 @@
+order: 9
+id: word-paragraph-onadded-event
+name: On adding paragraphs
+description: 'Registers, triggers, and deregisters onParagraphAdded event that tracks the addition of paragraphs.'
```
Comment (samantharamon) [inline samples/word/25-paragraph/onadded-event.yaml:4]:
```suggestion
description: 'Registers, triggers, and deregisters the onParagraphAdded event that tracks the addition of paragraphs.'
```

File: samples/word/25-paragraph/onadded-event.yaml [hunk 11]
```diff
@@ -0,0 +1,128 @@
...
+        <section class="samples ms-font-m">
+            <h3>Try it out</h3>
+            <button id="register-event-handler" class="ms-Button">
+            <span class="ms-Button-label">Register event handler</span>
+          </button>
+            <button id="insert-paragraphs" class="ms-Button">
+              <span class="ms-Button-label">Insert paragraphs</span>
+          </button>
+            <p>
+                <label style="margin-left: 20px">Paragraph ID:</label>
```
Comment (samantharamon) [inline samples/word/25-paragraph/onadded-event.yaml:93]: Suggestion: Consider adding text before the paragraph field to notify the user to copy the paragraph ID from the console.

File: samples/word/50-document/manage-tracked-changes.yaml (hunk 4 above)
Comment (ElizabethSamuel-MSFT) [reply samples/word/50-document/manage-tracked-changes.yaml:154]: Reorganized and ready for your review

File: samples/word/25-paragraph/onadded-event.yaml (hunk 11 above)
Comment (ElizabethSamuel-MSFT) [reply samples/word/25-paragraph/onadded-event.yaml:93]: Added

**Review by samantharamon — APPROVED** [review]

**Review by jipyua — APPROVED** [review]

**Pull request merged**